frontend/src/components/layout/navigation/whatsnew/WhatsNewContent.tsx

```diff
@@ -1,10 +1,25 @@
+import { OutboundLink } from "react-ga";
+import {
+  getPremiumSubscribeLink,
+  isPremiumAvailableInCountry,
+} from "../../../../functions/getPlan";
+import { useProfiles } from "../../../../hooks/api/profile";
+import { useRuntimeData } from "../../../../hooks/api/runtimeData";
+import { useGaViewPing } from "../../../../hooks/gaViewPing";
 import styles from "./WhatsNewContent.module.scss";
 
 export type Props = {
   heading: string;
   description: string;
   image: string;
   videos?: Record<string, string>;
+  cta?: {
+    type?: "upgrade" | undefined;
+    target?: string;
+    content: string;
+    onClick?: () => void;
+    gaViewPing?: Parameters<typeof useGaViewPing>[0];
```

Comment:
`gaViewPing` doesn't seem to be used at the moment? (Though maybe it should be?)

Reply:
Just got rid of it for now.